tools/snitch/snitcher/docker/linux/amd64/Dockerfile

```diff
@@ -1,21 +1,12 @@
-FROM ekidd/rust-musl-builder AS builder
-
-ARG SRC_DIR=.
-
-# Add source code
-COPY $SRC_DIR/ ./
-
-# Fix permissions on source code.
-RUN sudo chown -R rust:rust /home/rust
-
-# Build app
-RUN cargo build --release
-
-# Build real container
-
-FROM alpine:latest
-RUN apk --no-cache add ca-certificates
-COPY --from=builder /home/rust/src/target/x86_64-unknown-linux-musl/release/snitcher \
-    /usr/local/bin/
-
+FROM ubuntu:18.04
+
+RUN apt-get update -y && \
+    apt-get install -y apt-utils ca-certificates curl gpg && \
+    curl https://ftp-master.debian.org/keys/archive-key-9-security.asc | gpg --dearmor > archive-key-9-security.gpg && \
+    cp ./archive-key-9-security.gpg /etc/apt/trusted.gpg.d/ &&\
+    echo "deb http://security.debian.org/debian-security stretch/updates main" >> /etc/apt/sources.list && \
+    apt-get update -y && \
+    apt-get install -y libssl1.0.2
```

Comment:
Why do you need an older openssl?

Reply:
This is one of the dependencies snitcher required.  And the new version is not working for Ubuntu 18.04.

Reply:
@darobs would you please help to explain the problem that libssl1.0.2 needed to use on ubuntu:18.04?  thanks.

Reply:
addressed by using matched openssl during build and run.